### PR TITLE
Added the error page component

### LINKED ISSUE #31: error 404 page
image

don't add the gray circle

the 404 text should not be a part of the graphics

### PR DESCRIPTION
**Pull request by darsh175223**
Added an image of the Error page in dark mode. Added text "404 Error" in the middle of the image.

- resolves #31

### CODE REVIEW

File: src/components/ErrorPageDark.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  return (
+    <div className="relative flex h-screen items-center justify-center">
+      <div className="relative h-full w-full">
+        <Image
+          className="object-contain"
+          src={errorImageDark}
+          alt="Error background"
+          fill
+          priority
+        />
```
Comment (stanleylew5) [inline src/components/ErrorPageDark.tsx]:
```suggestion
        <Image
          src={errorImageDark}
          alt="Error background"
          fill
          priority
        />
```

File: src/components/ErrorPageDark.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      </div>
+      <p className="font-sans-serif absolute -translate-y-16 text-[8vw] text-yellow-400 drop-shadow-[0_0_15px_rgba(250,204,21,0.8)]">
```
Comment (stanleylew5) [inline src/components/ErrorPageDark.tsx]:
```suggestion
      <p className="font-sans-serif absolute -translate-y-[10vh] text-[8vw] text-yellow-400 drop-shadow-[0_0_15px_rgba(250,204,21,0.8)]">
```

Comment (stanleylew5) [reply src/components/ErrorPageDark.tsx]: use color from figma for yellow -- add it within tailwind config file

**Review by stanleylew5 — CHANGES_REQUESTED** [review]
first, replace the webp file with an SVG file, you can do so by copying the elements you want from the figma as an SVG and then creating a SVG file and then copying in the text into the SVG file.

make sure you remove your webp files in the process

also call the component within not-found.tsx

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**